zulip_bots/setup.py

```diff
@@ -3,14 +3,24 @@
 
 from __future__ import print_function
 
+import os
 import sys
 
 import generate_manifest
 
+version_py = os.path.join(os.path.dirname(__file__), "zulip_bots", "__init__.py")
+version_symlink = os.path.join(os.path.dirname(__file__), 'version.py')
+if os.path.islink(version_symlink):
```

Comment:
Because if another link was created during a previous run of setup.py, this code will fail very loudly and the setup won't succeed, which disrupts the flow of the release script in a there's-no-going-back-from-this fashion. In that case we should remove the pre-existing link and start anew! :)